conclave-auction/enclave/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 dependencies {
     implementation project(":common")
     implementation "com.r3.conclave:conclave-enclave"
-    testImplementation "com.r3.conclave:conclave-testing"
+    testImplementation "com.r3.conclave:conclave-host"
```

Comment:
You need this only if you want to add test case you your enclave class. We use EnclaveHost.load in the test class to load the enclave in mock mode. Since there isn't any test case written for this enclave module, I would suggest you to remove this.

Reply:
I see, thank you! OK I will do.
Do you want me to update this PR too?

Reply:
you can remove this in the same PR, I will merge it then.

Reply:
Thank you, @snedamle . I made a second commit to remove that line as suggested.